ui/app/AppLayouts/Chat/ContactsColumn/Channel.qml

```diff
@@ -165,12 +165,12 @@ Rectangle {
           wrapper.hovered = false
         }
         onClicked: {
+            chatsModel.setActiveChannelByIndex(index)
+            chatGroupsListView.currentIndex = index
```

Comment:
Because of what I've mentioned here: https://github.com/status-im/status-desktop/pull/1722/files#r561939780, I decided to move the function call up here (before the menu is opened).

This ensures we still have the right active channel selected when opening the menu. The downside is however, that opening the context menu will change the active channel as well.

We probably want to do some refactoring to address https://github.com/status-im/status-desktop/pull/1722/files#r561939780 properly so we don't have to do this here.

Reply:
We could either add a new `focusedChannel` similar to what we do in the Wallet view, or even better, just create a `channel` object to send it instead of sending `activeChannel`.

Like so:
```qml
channelContextMenu.openMenu({name: name, ...}, index)
```
Since we already have all the desired props, it should work